src/components/html/EcAdminSettingsForm.html

```diff
@@ -1,73 +1,85 @@
+        </div>
+      </div>
+      <b-collapse 
+        class="p-4 border-right border-left" 
+        accordion="my-accordion" 
+        :id="'accordion-' + i"
+      >
+        <div 
+          v-for="(config) in fields" 
           :key="`${Boolean(config.hide).toString()}-${config.field}`"
         >
-          <template v-if="checkNestedObjectsArray(config.schema)">
-            <div
-              v-for="({ dataList, lowerCaseTitle, field }) in [{
-                ...config,
-                dataList: config.hide ? hiddenData[config.field] : data[config.field],
-                lowerCaseTitle: (config.schema.items.title || '').toLowerCase()
-              }]"
-              key="nested"
+          <div 
+            class="alert alert-info" 
+            v-if="description"
+          >
+            <span 
+              slot="message" 
             >
-              <a-button
-                type="dashed"
-                class="mr-2 mb-1"
-                html-type="button"
-                @click.prevent="dataList.push({})"
-              >
-                {{ `${i19add} ${lowerCaseTitle}` }}
-              </a-button>
-
-              <a-upload
-                class="mb-1"
-                :before-upload="file => uploadCsv(dataList, file)"
-                accept=".csv"
-              >
-                <a-button
-                  type="dashed"
-                  html-type="button"
+              {{ description }}
+            </span>
+          </div>
+          <template v-if="checkNestedObjectsArray(config.schema)">
+            <div 
+              v-for=
+                "({ dataList, lowerCaseTitle, field }) in [{
+                  ...config,
+                  dataList: config.hide ? hiddenData[config.field] : data[config.field],
+                  lowerCaseTitle: (config.schema.items.title || '').toLowerCase()
+                }]" 
+              key="nested">
+              <div class="d-flex justify-content-start overflow-hidden">
+                <button 
+                  class="mr-2 btn btn-secundary btn-sm" 
+                  @click.prevent="dataList.push({})"
                 >
-                  <i class="fas fa-upload mr-2"></i>
-                  {{ i19upload }}
-                </a-button>
-              </a-upload>
+                  {{ `${i19add} ${lowerCaseTitle}` }}
+                </button>
+                <div class="btn btn-secundary btn-sm">
+                  <input 
+                  type="file"
+                  accept=".csv"
+                  id="upload"
+                  >
+                  <i class="fas fa-upload mr-2" style="z-index: 1"></i>
+                </div>
+                
+                  
+                  
+              </div>
 
-              <a-list
-                :dataSource="dataList"
-                class="mt-3 mb-4"
-                bordered
+              <div 
+                :dataSource="dataList" 
+                class="mt-3 mb-4 list-group"
               >
```

Comment:
```suggestion
              <div class="mt-3 mb-4 list-group">
```

`:data-source` era uma prop do `<a-list>`...